src/common/platform_function.h

```diff
@@ -85,4 +85,11 @@ void * FRTest_MemoryAlloc( size_t size );
  */
 void FRTest_MemoryFree( void * ptr );
 
+/**
+ * @brief To generate random number in INT format.
+ *
+ * @return A random number.
+ */
+int FRTest_GenerateRandInt();
```

Comment:
Might consider using uxRand() or xApplicationGetRandomNumber() instead of defining your own interface here.

Reply:
I think this is the defination in demos of amazon-freertos or FreeRTOS.
But uxRand() or xApplicationGetRandomNumber() are not requested to define in users' applications.
So we can define one in integration repo.
Of course users can redefine or wrap the function name to solve this problem.